scripts/npm-postversion-for-bower.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
Two of these Bower versioning scripts were removed and one was greatly simplified in the `windows` branch of the generator. That'll be the case for 2.0.0 of the generator; so, maybe it makes sense to run that version to get the simpler Bower versioning setup?